Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.cpp

```diff
@@ -250,7 +250,7 @@ namespace AZ
 
             for (const VkDeviceQueueCreateInfo& queueInfo : queueCreationInfo)
             {
-                delete queueInfo.pQueuePriorities;
```

Comment:
This is allocated with `new float[familyProperties.queueCount];` on [line 129](https://github.com/o3de/o3de/blob/ea785a9e5d7ec4d486d13d225eec6a3abc0c103d/Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.cpp#L129).